graylog2-web-interface/src/views/components/ViewActionsMenu.tsx

```diff
@@ -67,21 +81,28 @@ const ViewActionsMenu = ({ view, isNewView, metadata }) => {
 
   return (
     <ButtonGroup>
+      {showSaveButton && (
       <Button onClick={() => onSaveView(view)}
               disabled={isNewView || hasUndeclaredParameters || !allowedToEdit}
-              data-testid="dashboard-save-button">
```

Comment:
Please be aware that changes like this could also affect our [e2e-tests](https://github.com/Graylog2/e2e-tests). In some cases we introduced identifiers like this to be able to address elements, either in unit or in e2e tests. There are better ways to do this than test ids (like using a title), we just need to coordinate changes. In this case it was only needed for the unit tests afaics, so we are all good. I am working on integrating the e2e-tests into our regular build, so it will be more obvious in the future when the related change in the tests is missing (because the build will fail).

Reply:
Got it was worried about this but ran all the tests, didn't think that the e2e tests could be using it.